mypyc/codegen/emitmodule.py

```diff
@@ -918,7 +920,13 @@ def generate_module_def(self, emitter: Emitter, module_name: str, module: Module
 
         emitter.emit_line('return {};'.format(module_static))
         emitter.emit_lines('fail:',
-                           '{} = NULL;'.format(module_static),
+                           'Py_XDECREF({});'.format(module_static),
```

Comment:
Do `Py_CLEAR` for these, which will do an Py_XDECREF and NULL out the variable. (By virtue of being a macro that does not behave like a function...)